From: Alexander Larsson <alexl@redhat.com>
Date: Mon, 29 Aug 2016 11:46:33 +0200
Subject: Update: Don't double-call flatpak_dir_cleanup_removed

This is already called automatically from flatpak_dir_deploy_update.
In particular, in the system-installed case this will cause
a warning due to not being able to remove the old checkout with
permission denied.

Origin: upstream, 0.6.10, commit:874a67c721a5987eb803822fec9618621c4ff502
---
 app/flatpak-builtins-update.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/app/flatpak-builtins-update.c b/app/flatpak-builtins-update.c
index 136295f..77de3d5 100644
--- a/app/flatpak-builtins-update.c
+++ b/app/flatpak-builtins-update.c
@@ -299,8 +299,6 @@ flatpak_builtin_update (int           argc,
       return FALSE;
     }
 
-  flatpak_dir_cleanup_removed (dir, cancellable, NULL);
-
   if (failed)
     return flatpak_fail (error, _("One or more updates failed"));
 
